test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -113,13 +112,7 @@ func initProperties(properties map[string]string) error {
 		IBMCloudProps.IsSelfManaged = true
 	}
 
-	debugStr := os.Getenv("DEBUG")
-	if strings.EqualFold(debugStr, "yes") || strings.EqualFold(debugStr, "true") {
-		IBMCloudProps.IsDebug = true
-	}
-	if IBMCloudProps.IsDebug {
-		fmt.Printf("%+v\n", IBMCloudProps)
-	}
```

Comment:
Could we replace this with and environment variable and IBMCloudProps (which would likely be share across the other environments to control the log level with `log_level = info` as the default?